src/plugins/intel_cpu/src/nodes/reorder.cpp

```diff
@@ -447,19 +474,20 @@ void Reorder::reorderData(const Memory &input, const Memory &output) {
                 cpu_convert(data, tmpBuff.data(), DnnlExtensionUtils::DataTypeToIEPrecision(input.GetDataType()),
                             outPrc, input.GetSize() / input.getDesc().getPrecision().size());
 
-                Memory tmpMem(output.getEngine());
+                Memory tmpMem(engine);
                 auto tmpDesc = input.getDesc().cloneWithNewPrecision(outPrc);
                 tmpMem.Create(std::move(tmpDesc), tmpBuff.data());
 
-                pReorder = std::unique_ptr<dnnl::reorder>(new dnnl::reorder(tmpMem.GetPrimitive(), output.GetPrimitive()));
                 srcMemory = tmpMem.GetPrimitive();
-            } else {
-                throw;
+                pReorder = getReorder(cache, srcMemory, dstMemory);
+            }
+            if (!pReorder) {
+                IE_THROW() << "Not supported reorder conversion, in type: " <<
+                    input.getDesc().getPrecision() << ", out type: " << output.getDesc().getPrecision();
```

Comment:
If at the moment reorder has not been created, that means there is no available reordering for the specific tensors, but not on purpose of different precisions, the precisions were aligned by the `cpu_covert` call above. So we probably need to change the error message to : "No reorder available for the following tensor descriptors: " << input.getDesc().serializeFormat() << " and " << output.getDesc().serializeFormat();

Reply:
Updated.